PRS4001-5000/PRS4711Giyorgis.xml

```diff
@@ -44,41 +44,46 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="SG" when="2017-02-13">addition</change>
          <change who="DR" when="2017-10-11">Added Arabic name</change>
+         <change when="2024-09-02" who="CH">Updated titles and dates</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName xml:lang="gez" xml:id="n1" type="birth">ጊዮርጊስ፡ ወልደ፡ ዓሚድ፡</persName>
-               <persName xml:lang="gez" corresp="#n1" type="normalized">Giyorgis Walda ʿAmid</persName>
+               <persName xml:lang="gez" xml:id="n1" type="birth">ጊዮርጊስ፡ ወልደ፡ አሚድ፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">Giyorgis Walda ʾAmid</persName>
                <persName xml:lang="gez" xml:id="n2" type="alt">ጊርጊስ፡ አልመኪን፡</persName>
                <persName xml:lang="gez" corresp="#n2" type="normalized">Ǧirǧis al-Makīn</persName>
-               <persName xml:lang="ar" xml:id="n3" type="alt">جرجس المكين بن العميد</persName>
-               <persName xml:lang="ar" corresp="#n3" type="normalized">Ǧirǧis al-Makīn b. al-ʿAmīd</persName>
-               
+               <persName xml:lang="ar" xml:id="n3" type="alt">جرجس المكين ابن العميد</persName>
+               <persName xml:lang="ar" corresp="#n3" type="normalized">Ǧirǧis al-Makīn ibn al-ʿAmīd</persName>
+               <occupation type="political">Served in the military administration and was a member of the Dīwān al-ǧayš of various Ayyubid 
+                  and Mamluk sultans</occupation>
                <occupation type="literary">historian</occupation>
                <nationality type="Egypt"/>
                <faith type="Coptic"/>
                <birth when="1206">He was born in <date>1206</date> at <placeName ref="wd:Q85">Cairo</placeName>.</birth>
-               <floruit notBefore="1206" notAfter="1273">He was a Christian Arab historian whose works were incorporated into Ethiopian tradition. 
-                  He wrote the  book <ref type="work" corresp="LIT4723TarikaWaldaAmid">al-Maǧmaʿ al-mubārak</ref> in <date> between 1262-1268 </date>
-                  which was translated during the reign of  
-                  <persName ref="PRS6229LebnaDe">
-                     <roleName type="title">ʾaṣe</roleName>  Lǝbna Dǝngǝl</persName>
-
-                  as <foreign xml:lang="gez">ታሪከ፡ ወልደ፡ ዓሚድ</foreign>. 
-                  
+               <floruit notBefore="1206" notAfter="1293">He was a Christian Arab historian whose work was translated and incorporated into 
+                  Ethiopian tradition. 
+                  He wrote his <ref type="work" corresp="LIT4723TarikaWaldaAmid">Tārīḫ</ref> in <date> between 1262-1268</date>,
+                  which was translated twice during the reign of 
+                  <persName ref="PRS10626ZaraY"><roleName type="title">ʾaṣe</roleName>Zarʾa Yāʿqob</persName> and 
```

Comment:
very nice, just add space before Zar'a (or it will be printed ʾaṣeZarʾa)

Reply:
Yes. This branch has already been merged and I deleted it from my github-Desctop. I will do the change in the next pull request to come.